### PR TITLE
[ASDK Management] PR / Issue Checklists

### PR DESCRIPTION
**Pull request by hannahmbanana**
Provide checklists for new Pull Requests and Issues via https://help.github.com/articles/helping-people-contribute-to-your-project/ 

I went a little overkill with the PR one. Let me know which ones you think are useful or not. 

Should we also do a checklist for new issues or just PRs?

@maicki @Adlai-Holler

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+### Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+- [ ] I have [signed the CLA](https://code.facebook.com/cla) right now or previously.
+- [ ] I have read the [**CONTRIBUTING** document](https://github.com/facebook/AsyncDisplayKit/blob/master/CONTRIBUTING.md.
+- [ ] I have titled my pull request in the style of other PRs, e.g. "[ASTextNode] Fix..."
+- [ ] My change requires updates to the AsyncDisplayKit.org documentation.
+- [ ] All new and existing tests passed.
```
Comment (Adlai-Holler) [inline .github/PULL_REQUEST_TEMPLATE.md]: The tests passed line is more trouble than its worth. GitHub will make it really clear whether the tests have passed for each commit, and they may pass/fail/pass/fail between different commits.

**Comment by Adlai-Holler** [comment]
Aside from the one note, I'd say 💯

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (hannahmbanana) [inline .github/PULL_REQUEST_TEMPLATE.md]: New plan: let's implement an Issues template (but make it very approachable / nice as possible). Issue volume > PR volume.  ASDK version, backtraces, screenshot, code snippet, sample project. ===> We will get back faster to you!  Simple > comprehensive! 

For PRs, whoever merges it should try to include this info in whatever format we want. Should we require a sample project?

**Comment by hannahmbanana** [comment]
@levi: Notes / formal documentation from our GitHub maintainers meeting last week.

**Comment by hannahmbanana** [comment]
@maicki: This is ready to merge I believe. Let me know when do so that I can check that the ISSUE template works.

**Pull request merged**